config/javascript.js

```diff
@@ -42,22 +41,20 @@ gulp.task('js:vendor', function() {
  * Watch scripts for changes and move to the "static" folder
  */
 
-gulp.task('js:watch', function() {
-  gulp.watch('src/js/**/*.js', ['js:concat', 'js:vendor']);
+gulp.task('js:watch', async function() {
+  gulp.watch('src/js/**/*.js', gulp.series('js:concat', 'js:vendor'));
 });
 
 gulp.task('js:dist', function() {
   return gulp.src('static/js/rivet.js').pipe(gulp.dest('./js'));
 });
 
-gulp.task('js:header', function() {
-  gulp
-    .src('./js/rivet.js')
+gulp.task('js:header', async function() {
```

Comment:
Same comment as above here. I could see the benefit of just using regular old callbacks consistently instead of sprinkles of async code. Just an idea though, maybe a little nitpick-y 😄